src/components/dashboard/main-content/course-enrollments/CourseAssignmentAlert.jsx

```diff
@@ -6,6 +6,7 @@ import { Info } from '@edx/paragon/icons';
 import { getContactEmail } from '../../../../utils/common';
 
 const CourseAssignmentAlert = ({
+  showAlert,
```

Comment:
Good call to make this alert component act as a controlled component instead of uncontrolled.